### PR TITLE
[b/377657866] Remove ClouderaMetadataConnector from the code

### PR DESCRIPTION
**Pull request by vladislav-sidorovich**
There is a decision do not use Metaconnectors and use plain single connectors instead. 
The task is to clean-up Dumper code.

### CODE REVIEW

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/HadoopInitializerTask.java [hunk 1]
```diff
@@ -35,10 +34,10 @@ public HadoopInitializerTask() {
   @Override
   protected Void doRun(TaskRunContext context, @Nonnull ByteSink sink, @Nonnull Handle handle)
       throws Exception {
-    MetaHandle metaHandle = (MetaHandle) handle;
-    metaHandle.initializeConnector(
-        ClouderaMetadataConnector.CONNECTOR_NAME,
-        new ConnectorArguments("--connector", ClouderaMetadataConnector.CONNECTOR_NAME));
+
+    String connectorName = ClouderaMetadataConnector.CONNECTOR_NAME;
+    ConnectorArguments connectorArguments = new ConnectorArguments("--connector", connectorName);
```
Comment (sayuzbas-google) [inline dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/HadoopInitializerTask.java:39]: nit: can directly use `ClouderaMetadataConnector.CONNECTOR_NAME`

**Review by sayuzbas-google — APPROVED** [review]

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/DumpMetadataTask.java [hunk 2]
```diff
@@ -52,20 +49,17 @@ public DumpMetadataTask(@Nonnull String format) {
...
-            ? connectors.stream().map(UnderlyingConnector::name).collect(toImmutableList())
-            : null;
+    this.connectors = connectors != null ? ImmutableList.copyOf(connectors) : null;
   }
 
   public static DumpMetadataTask create(
       @Nonnull ConnectorArguments arguments,
       @Nonnull String format,
-      @Nonnull ImmutableList<UnderlyingConnector> connectors) {
+      @Nonnull ImmutableList<String> connectors) {
```
Comment (misolt) [inline dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/DumpMetadataTask.java:62]: Optional: `connectors` -> `connectorNames`

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/HdfsInitializerTask.java [hunk 3]
```diff
@@ -36,10 +35,10 @@ public HdfsInitializerTask() {
...
   protected Void doRun(TaskRunContext context, @Nonnull ByteSink sink, @Nonnull Handle handle)
       throws Exception {
-    MetaHandle metaHandle = (MetaHandle) handle;
-    metaHandle.initializeConnector(
-        HdfsExtractionConnector.CONNECTOR_NAME,
-        new ConnectorArguments("--connector", HdfsExtractionConnector.CONNECTOR_NAME));
+
+    String connectorName = HdfsExtractionConnector.CONNECTOR_NAME;
+    ConnectorArguments connectorArguments = new ConnectorArguments("--connector", connectorName);
+    // todo use in Hadoop
```
Comment (rocgranada) [inline dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/HdfsInitializerTask.java:41]: What does this mean? The HDFS connector is run separatedly from the Hadoop connector, right?

Comment (vladislav-sidorovich) [reply dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/HdfsInitializerTask.java:41]: Yes, HDFS connector is run separately from the Hadoop connector in the current version of code. However, as I see it's just empty.
So, I leave this comment to do design decision in the scope of the following refactor task (Hadoop related).

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/HiveInitializerTask.java [hunk 4]
```diff
@@ -36,11 +35,10 @@ public HiveInitializerTask() {
...
       throws Exception {
-    MetaHandle metaHandle = (MetaHandle) handle;
-    metaHandle.initializeConnector(
-        HiveMetadataConnector.CONNECTOR_NAME,
-        new ConnectorArguments(
-            "--connector", HiveMetadataConnector.CONNECTOR_NAME, "--assessment"));
+    String connectorName = HiveMetadataConnector.CONNECTOR_NAME;
+    ConnectorArguments connectorArguments =
+        new ConnectorArguments("--connector", connectorName, "--assessment");
+    // todo use in Hadoop
```
[the remark this answers is not in the document]
Comment (vladislav-sidorovich) [reply dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/HiveInitializerTask.java:41]: make sense

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/HadoopInitializerTask.java (hunk 1 above)
Comment (vladislav-sidorovich) [reply dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/HadoopInitializerTask.java:39]: The part of code just do nothing and will be rewritten in the next task. So, I think it's ok to leave it as is.

File: dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/DumpMetadataTask.java (hunk 2 above)
Comment (vladislav-sidorovich) [reply dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/DumpMetadataTask.java:62]: Well, looks like it should be completely removed. Removed.

**Pull request merged**